ubuntudesign/www.canonical.com
# Update all taleo links

Most taleo links currently follow 2 redirects and eventually end up on ldd.tbe.taleo.net.

This is inefficient, but more importantly it makes it much more complicated to allow these requests through the firewall.

So I'm updating all these to point to their eventual targets.

QA
---

Go through each page on the `/careers` section and check that all feeds look correct and that all links to search for jobs still work.
